### ISSUE TITLE
[BUG] [Formatter] blank-line-after-tag does not match tags with Nunjucks whitespace-control dashes ({%- -%})

### ISSUE THREAD

**Issue by jameswragg** [djlint/djLint#2102]
## Bug

`--blank-line-after-tag` (and `blank_line_after_tag` in config) silently skips template tags that use Nunjucks/Jinja2 **whitespace-control dashes** (`{%-` / `-%}`).

## Cause

The regex used in `formatter/condense.py` is:

```python
rf"((?:{{%\s*?{tag.strip()}\b[^}}]+?%}}\n?)+)"
```

`{%\s*?` expects only whitespace after `{%`, so a leading dash (`{%-`) breaks the match. Trailing `-%}` is also not handled (the pattern ends with `%}`).

## Steps to reproduce

```njk
{%- macro Foo() -%}
  <div>foo</div>
{%- endmacro -%}
<p>no blank line inserted above (bug)</p>
{% macro Bar() %}
  <div>bar</div>
{% endmacro %}
<p>blank line correctly inserted above</p>
```

```sh
djlint --reformat --profile=nunjucks --blank-line-after-tag endmacro input.njk
```

**Actual output:** blank line added after `{% endmacro %}` only — `{%- endmacro -%}` is unaffected.  
**Expected output:** blank line added after **all** `endmacro` tags regardless of whitespace-control dashes.

## Suggested fix

Change the regex to accommodate an optional leading/trailing dash:

```python
rf"((?:{{%-?\s*?{tag.strip()}\b[^}}]+?-?%}}\n?)+)"
```

## Environment

- djlint version: 1.39.2
- Profile: nunjucks (also affects jinja/django where `{%-` is valid)
